python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -116,6 +118,7 @@ Parameters to use when loading a file to a Pandas dataframe
        :start-after: [START load_file_example_6]
        :end-before: [END load_file_example_6]
 
+#. **load_options** - `load_options`_
```

Comment:
Same here; the description is not helping to understand what these options are. It is probably worth to also say that the `native_support_kwargs` will be replaced by this